src/interfaces/IMetaMorpho.sol

```diff
@@ -95,8 +95,12 @@ interface IMetaMorphoBase {
     /// @notice Revokes the pending cap of the market defined by `id`.
     function revokePendingCap(Id id) external;
 
-    /// @notice Submits a forced market removal from the vault, potentially losing all funds supplied to the market.
+    /// @notice Submits a forced market removal from the vault, losing all funds supplied to the market.
```

Comment:
I think we should keep "potentially" as there might be a chance that force removed market can be added again, and in that case the funds aren't lost.

It doesn't work if the market is doomed forever, but if working markets are removed (because of a malicious curator), the funds are not definitely lost.

Reply:
https://github.com/morpho-org/metamorpho/pull/354#discussion_r1433753164